app/src/main/java/com/bunbeauty/papakarlo/feature/profile/screen/profile/ProfileFragment.kt

```diff
@@ -350,6 +349,55 @@ class ProfileFragment : BaseFragmentWithSharedViewModel(R.layout.layout_compose)
         }
     }
 
+    @Composable
+    private fun OrderProfile(
```

Comment:
- `shape = RoundedCornerShape(0.dp)`
- should have no divider
- should have bottom padding
- and divider under bottom space